File: problems/easy/tax_error/tax_error.cpp
#include <iostream>
using namespace std;

double calculateTax(double income) {
    if (income < 0) {
        return 0; 
    } 
    if (income == 0) {
        return 0;
    }
    double taxRate = 0.15;
    
    return income * taxRate; 
   
}

int main() {
    cout << calculateTax(10000) << endl; 
    return 0;
}

File: problems/easy/tax_error/tax_error.js
function calculateTax(income) {
    if (income < 0) {
      return "Income can't be negative";  
    }
    const taxRate = 0.15;
    let tax = 0;
    income * taxRate;  
    return tax;
  }
  
  console.log(calculateTax(10000));  